source/docs/appendix/gallery.rst

```diff
@@ -8,6 +8,12 @@ Collection of robot designs from past seasons. Feel free to borrow some design s
 2023-2024 Center Stage
 ------------------------
 
+23396 hivemind
```

Comment:
The teams within a given season are sorted in numerical order, and so hivemind should be last out of the teams present for Center Stage.